tools/build/ThirdPartyNotices.md

```diff
@@ -13,3 +13,27 @@ Permission is hereby granted, free of charge, to any person obtaining a copy of
 The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+## [recursive-readdir](https://github.com/jergason/recursive-readdir)
+
+The MIT License (MIT)
+
+Copyright (c) \<year> \<copyright holders>
```

Comment:
is this right? @wgoehrig

Reply:
@aruniverse if the question is "do we have to reproduce their license file verbatim even if it is silly?" The answer is in the notice itself. It says that 

```txt
The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.
```
We must just copy whatever it says without changes to comply.